FreeRTOS-Plus/Demo/MQTT_Mutual_Auth_Qemu_mps2/core_mqtt_config.h

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
This file seems to duplicate others in this repo. Does it really need to be copied here?

Reply:
It was included in each of the coreMQTT windows simulator demos which is why I also included it here to follow that structure